docs/analytics/apis/user-privacy-api.md

```diff
@@ -11,6 +11,12 @@ The User Privacy API helps you comply with end-user data deletion requests manda
 
 --8<-- "includes/auth-basic.md"
 
+!!!note "Authorization in cURL"
```

Comment:
Sorry for missing this note in the ticket, but I believe this applies to all communication methods, which specify the header directly, not only cURL

I would recommend a more generic statement (feel free to use different wording =)):
```
!!!note "Setting Authorization Header"

When specifying the `Authorization: Basic` header directly, the `acesss_key:secret_key` credentials pair should be Base64-encoded. It should look something like this:
```

Reply:
🤦 yes. I was not firing on all cylinders yesterday, thank you.